src/main/scala/it/unibo/authsim/client/app/mvvm/model/security/SecurityPolicy.scala

```diff
@@ -21,7 +21,7 @@ object SecurityPolicy:
       def descriptor(defaultVal: DefaultVal) =
         s"""${base}\n
             ${if defaultVal.policy.transmissionProtocol.isDefined then s"Credentials are trasmitted with protocol ${defaultVal.policy.transmissionProtocol.get.getClass.getSimpleName.toUpperCase}." else "No protocol." }\n
-            The alphabet:  ${defaultVal.policy.credentialPolicies.map(_.alphabet).map(a => a.lowers.concat(a.uppers).concat(a.digits).concat(a.symbols).mkString).distinct.mkString}"""
+            The alphabet:  ${defaultVal.policy.credentialPolicies.map(_.alphabet).map(a => a.alphanumericsymbols.mkString).distinct.mkString}"""
```

Comment:
With the fact that you use `Set`, the `alphanumericsymbols `is not in order

Reply:
This could be resolved by using `OrderedSet`s when the order is needed.

As of now, I'm supposing that `Policy` related stuff needs an order, so I'll change them to use an `OrderedSet`.